code/repo_sync

```diff
@@ -294,10 +295,7 @@ def curl(url, destinationpath, onlyifnewer=False, etag=None, resume=False):
         print >> fileobj, 'fail'           # throw error if download fails
         print >> fileobj, 'dump-header -'  # dump headers to stdout
         print >> fileobj, 'speed-time = 30' # give up if too slow d/l
-        # next line does not work on windows
-        #print >> fileobj, 'output = "%s"' % tempdownloadpath
-
-        print >> fileobj, 'ciphers = HIGH,!ADH' # use only >=128 bit SSL
+        print >> fileobj, 'tlsv1'          # use only TLS 1.x
```

Comment:
Quick testing seems to indicate this change doesn't explode with curl on 10.6, which the oldest thing reposado was ever supported on that I can still make run on hardware I have access to...